src/Stryker.Core/Stryker.Core.UnitTest/Initialisation/InputFileResolverTests.cs

```diff
@@ -153,6 +153,131 @@ public void InputFileResolver_InitializeShouldResolveImportedProject()
             result.ProjectContents.GetAllFiles().Count().ShouldBe(3);
         }
 
+        [Fact]
+        public void InputFileResolver_InitializeShouldNotResolveImportedPropsFile()
+        {
+            string sourceFile = File.ReadAllText(_currentDirectory + "/TestResources/ExampleSourceFile.cs");
+
+            string projectFile = @"
+<Project Sdk=""Microsoft.NET.Sdk"">
+    <PropertyGroup>
+        <TargetFramework>netcoreapp2.0</TargetFramework>
+        <IsPackable>false</IsPackable>
+    </PropertyGroup>
+
+    <ItemGroup>
+    </ItemGroup>
+               
+     <Import Project=""../SharedProject/Example.props"" Label=""Shared"" />
+
+</Project>";
+
+            var fileSystem = new MockFileSystem(new Dictionary<string, MockFileData>
+                {
+                    { Path.Combine(_filesystemRoot, "SharedProject", "Example.props"), new MockFileData("")},
```

Comment:
This is misleading. The folder "SharedProject" does not contain a sharedproject in this case :)